### PR TITLE
async support for "Channel" object - aenter and aexit function

### PR DESCRIPTION
**Pull request by lsching17**
For #7

### CODE REVIEW

**Comment by lsching17** [comment]
> Can you add a simple test case (or modify an existing test) for the new context manager interfaces?

The file is "tests\testclient.py"?

**Comment by hubo1016** [comment]
@lsching17 yes.

**Comment by lsching17** [comment]
PR updated

File: aiogrpc/channel.py [hunk 1]
```diff
@@ -327,6 +327,12 @@ def __init__(self, _channel, loop=None, executor=None, standalone_pool_for_strea
         self._standalone_pool = standalone_pool_for_streaming
         self._subscribe_map = {}
 
+    async def __aenter__(self):
+        return
```
[the remark this answers is not in the document]
Comment (lsching17) [reply aiogrpc/channel.py]: changed

**Comment by lsching17** [comment]
> @lsching17 yes.

unit test added

**Comment by lsching17** [comment]
> Can you add a simple test case (or modify an existing test) for the new context manager interfaces?

unit test added

**Pull request merged**